### DISCUSSION TITLE
django api + flet

### DISCUSSION THREAD

**Discussion by p17m0** [flet-dev/flet#1817] [General]
Hello everyone!
I'm interested about using "requests" in flet to get data from my api. Will it work after packing?
Because i want to make ios and android apps.
Or i have to use only flet?

**Comment by FeodorFitsner** [comment]
Yes, `requests` and `httpx` (async variant) are working on iOS and Android as they are "pure" Python (with no C/Rust code) libraries.

**Reply by p17m0** [reply]
Thank you very much!
Flet is my passion.
I really appreciate!
